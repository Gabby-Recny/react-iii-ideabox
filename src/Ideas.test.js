import React from 'react';
import { shallow } from 'enzyme';
import Ideas from './Ideas';

describe('Ideas', () => {
  let wrapper;
  let mockIdeas;
  let mockDeleteIdea;

  beforeEach(() => {
    mockIdeas = [
      {id: 1, title: 'fanny pack for holding throwing knives', description: 'on-the-go deadliness'}
    ];
    mockDeleteIdea = jest.fn();
    wrapper = shallow(<Ideas ideas={mockIdeas} deleteIdea={mockDeleteIdea} />);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
