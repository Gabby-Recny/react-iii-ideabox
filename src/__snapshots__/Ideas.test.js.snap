// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ideas should match snapshot 1`] = `
<div
  className="ideas-container"
>
  <Card
    deleteIdea={[MockFunction]}
    description="on-the-go deadliness"
    id={1}
    key="1"
    title="fanny pack for holding throwing knives"
  />
</div>
`;
